test/app/models/Cinematic.spec.js

```diff
@@ -1,5 +1,19 @@
 /* eslint-env jasmine */
-import { getLeftCharacterThangTypeSlug, getRightCharacterThangTypeSlug } from '../../../app/schemas/selectors/cinematic'
+import {
+  getLeftCharacterThangTypeSlug,
+  getRightCharacterThangTypeSlug,
+  getLeftHero,
+  getRightHero,
+  getClearBackgroundObject,
+  getBackgroundObject,
+  getBackground,
+  getClearText,
+  getSpeaker,
+  getBackgroundSlug,
+  exitCharacter,
```

Comment:
@shubhi1092 I responded above, saying that I was only exporting the `get<selector>` but then I broke my own naming rules here!!! :<
Will fix